### ISSUE TITLE
Tensor initialization as seed for inference

### ISSUE THREAD

**Issue by carrascomj** [carrascomj/shapels#5]
Creation Ops: https://docs.pytorch.org/docs/stable/torch.html#tensor-creation-ops

Tensor creation ops should be implemented as base cases to establish the shape of a tensor, in the same way that jaxtyping hints or, even closer, shape unrolling work.

**Comment by carrascomj** [comment]
Closed by d7ef533e6a3764683eccb071f749e0ecb43f78bf

There are still some exotic creation functions (i.e., I rarely use them) and the `torch.Tensor.new_*` methods to be implemented but will be added on a case by case basis if there is interest.

(With that said, I don't foresee adding creation ops from non-tensor data, like `torch.tensor`.)
